Fix more typos

anne-glerum opened the pull request:
Fix the typos I identified for the workshop yesterday. Some of them have been fixed by the others, so let's wait till those PRs are merged.

* [x] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

anne-glerum committed:
Fix typos

anne-glerum committed:
Also capitalize Darcy

bangerth commented:
Do you want to merge this PR, or are you still waiting for other PRs to get merged first?

anne-glerum commented:
@bangerth I've rebased again and updated the parameter documentation. I see one problem though. Because I compiled this branch against FastScape, the parameter documentation now also includes FastScape parameters. Should I just revert those additions? (I'll be working on including FastScape in the tester soon.)

gassmoeller commented:
/rebuild

gassmoeller commented:
Very nice, ready to merge when the testers are done.

gassmoeller commented:
Oh, I didnt read your post. Yes please remove the fastscape parameters for now until we have a tester ready.

anne-glerum committed:
Update parameter documentation

anne-glerum commented:
Fastscape documentation has been removed. I don't think the failing tester is due to this PR.
